src/scripts.js

```diff
@@ -1,40 +1,106 @@
+const userRepo = new UserRepository(userData);
+// const currentUser = new User(userRepo.userList[0]);
```

Comment:
Are we using this or can this be removed?

Reply:
No I just commented it out because that's how I was originally using currentUser but then changed it to be able to use your generateRandomUser function